sessionctx/variable/sysvar.go

```diff
@@ -1801,7 +1801,7 @@ var defaultSysVars = []*SysVar{
 		s.NoopFuncsMode = TiDBOptOnOffWarn(val)
 		return nil
 	}},
-	{Scope: ScopeGlobal | ScopeSession, Name: TiDBReplicaRead, Value: "leader", Type: TypeEnum, PossibleValues: []string{"leader", "follower", "leader-and-follower", "closest-replicas", "closest-adaptive", "learner"}, SetSession: func(s *SessionVars, val string) error {
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBReplicaRead, Value: "leader", Type: TypeEnum, PossibleValues: []string{"leader", "prefer-leader", "follower", "leader-and-follower", "closest-replicas", "closest-adaptive", "learner"}, SetSession: func(s *SessionVars, val string) error {
```

Comment:
Don't forget to update the user document.

Reply:
> Don't forget to update the user document.

Yeap, thx. It's already recorded in the related prs: 
* docs: https://github.com/pingcap/docs/pull/12337 ;
* docs-cn: https://github.com/pingcap/docs-cn/pull/12865;